sentry_sdk/integrations/fastapi.py

```diff
@@ -62,6 +65,22 @@ def patch_get_request_handler():
 
     def _sentry_get_request_handler(*args, **kwargs):
         # type: (*Any, **Any) -> Any
+        dependant = kwargs.get("dependant")
+        if dependant and not asyncio.iscoroutinefunction(dependant.call):
+            old_call = dependant.call
+
+            def _sentry_call(*args, **kwargs):
+                # type: (*Any, **Any) -> Any
+                hub = Hub.current
+                with hub.configure_scope() as sentry_scope:
+                    if sentry_scope.profile is not None:
+                        sentry_scope.profile.active_thread_id = (
+                            threading.current_thread().ident
```

Comment:
Because `current_thread()` can be a dummy object (see https://docs.python.org/3/library/threading.html#threading.current_thread) maybe it is saver to use ` threading.get_ident()`?
(But if the dummy object also has a `.ident` my comment can be ignored)

Reply:
`_DummyThread` inherits from `Thread` ([source](https://github.com/python/cpython/blob/main/Lib/threading.py#L1437)) and so both should have the `ident` [property](https://docs.python.org/3/library/threading.html#threading.Thread.ident).